testsuite/MDAnalysisTests/coordinates/base.py

```diff
@@ -130,7 +131,10 @@ def __init__(self):
         self.jump_to_frame.positions = 2 ** 3 * self.first_frame.positions
         self.jump_to_frame.frame = 3
 
-        self.dimensions = np.array([80, 80, 80, 60, 60, 90], dtype=np.float32)
+        self.dimensions = np.array([81.1, 82.2, 83.3, 60, 60, 90],
```

Comment:
Ideally, the angles should be different in all dimensions too. It is extremely easy to forget to replace a `x` by a `y` in a copy paste and have `x` and `y` coupled by mistake. As most systems are isotropic or semi-isotropic, this is easy to miss.